### DISCUSSION TITLE
🚀 Feature Request: Add Undo/Redo buttons to the new Editor Top Bar

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#22559] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

To improve the workflow in the Elementor editor, it would be great if you could add redo and undo buttons to the top bar.

### Describe the solution you'd like

![undo-redo](https://github.com/elementor/elementor/assets/88034902/ad17cf6b-19e0-49aa-aefa-4ce9a5d323b9)

**Comment by WPMonkey** [comment]
Yes please! It's very unfortunate that this was removed from the interface. My keyboard shortcut for undo instead opens the last closed tab in the browser, so doesn't work that way for me.

**Comment by sara-kariv** [comment]
+1
